Lib/asyncore.py

```diff
@@ -182,8 +184,8 @@ def poll2(timeout=0.0, map=None):
 
```

Comment:
We can copy only the active objects here:

    if flags:
        pending[fd] = obj
        pollster.register(fd, flags)